tests/docker/Dockerfile

```diff
@@ -105,11 +105,11 @@ RUN mkdir -p "/opt/kafka-2.3.1" && chmod a+rw /opt/kafka-2.3.1 && curl -s "$KAFK
 # install go
 RUN mkdir -p /usr/local/go/ && \
     bash -c 'if [[ $(uname -m) = "aarch64" ]]; then ARCHID="arm64"; else export ARCHID="amd64"; fi && \
-    curl -sSLf --retry 3 --retry-connrefused --retry-delay 2 https://golang.org/dl/go1.17.linux-${ARCHID}.tar.gz | tar -xz -C /usr/local/go/ --strip 1'
+    curl -sSLf --retry 3 --retry-connrefused --retry-delay 2 https://golang.org/dl/go1.19.linux-${ARCHID}.tar.gz | tar -xz -C /usr/local/go/ --strip 1'
```

Comment:
```suggestion
    curl -sSLf --retry 3 --retry-connrefused --retry-delay 2 https://golang.org/dl/go1.19.2.linux-${ARCHID}.tar.gz | tar -xz -C /usr/local/go/ --strip 1'
```

use the latest patch release of go1.19 codeline to get the latest security fixes. see https://go.dev/doc/devel/release#go1.19